### DISCUSSION TITLE
Select more then one Element?

### DISCUSSION THREAD

**Discussion by svki-dev** [easepick/easepick#166] [General]
Hi,
I have used a datepicker in several places on the website. 
As it is now it doesn't work because document.getElementByID('datepicker') the first datepicker is selected. 
Is there a way with document.querySelecotrAll or something like this
![easepick](https://user-images.githubusercontent.com/105556377/217549897-eb28933b-88c5-469a-a387-5ad40ea01e86.png)
?

**Comment by cavasinf** [comment]
Can't you `forEach` a `querySelectorAll` BEFORE the `easypick.create`?

```ts
document.querySelectorAll('.easypick').forEach((element) => {
   const picker = new easypick.create({
      element: element,
      // Rest of the logic..
   })
}
```

**Note:** I don't think it is a good practice to have several same ids.
Better use a class to target..
